src/manager/lotus.rs

```diff
@@ -219,6 +219,34 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
         Ok(())
     }
 
+    async fn set_validator_net_addr(
+        &self,
+        subnet: SubnetID,
+        from: Address,
+        validator_net_addr: String,
+    ) -> Result<()> {
+        if !self.is_network_match(&subnet).await? {
```

Comment:
This call should go to the parent subnet of the subnetID for which the validator is being changed, I don't know if this check is correct in this case.

Reply:
resolved